wiki/Community/osu!monthly/en.md

```diff
@@ -0,0 +1,53 @@
+# osu!monthly
+
+**osu!monthly** is a monthly newsletter managed by the [osu!monthly Team](/wiki/People/osu!monthly_Team), with the goal of summarising the various community and official events, news, announcements, etc. that occur every month. The project has been led by several other groups of community members in the past, with the goal always being the same: inform the average player about the previous and current events in order to shed light over the entire osu! community.
+
+## Becoming an osu!monthly contributor
+
+osu!monthly is a community effort with many people contributing to its creation. Anyone that wants to help or contribute, in ways such as writing, sharing opinions and thoughts, or providing feedback, is always welcome to do so. The [osu!monthly Team](/wiki/People/osu!monthly_Team) consists of such contributors who consistently help out with the project.
+
+All discussion and project organisation takes place in the `#osu-news` channel of the [osu!dev Discord server](/wiki/Community/osu!dev_Discord_server).
+
+## History
+
+The original run of osu!monthly created by [jjrocks](https://osu.ppy.sh/users/30985), [Corin](https://osu.ppy.sh/users/639270), and friends ran from March 2011 to October 2011. The magazine was then released in a `.pdf` format with all the most important information about the osu! community.
```

Comment:
nit: `.pdf` is an extension, the format is PDF (which also expands to "portable document **format**", leading to the [RAS syndrome](https://en.wikipedia.org/wiki/RAS_syndrome))
```suggestion
The original run of osu!monthly created by [jjrocks](https://osu.ppy.sh/users/30985), [Corin](https://osu.ppy.sh/users/639270), and friends ran from March 2011 to October 2011. The magazine was then released in a PDF format with all the most important information about the osu! community.
```